src/routes/genreoverviewpage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GenreButton } from "../components/genre";
+import { genres, Genre } from "../utils/genre";
+import { Button } from "../components/button";
+import { IoIosArrowBack } from "react-icons/io";
+
+export function GenreOverviewPage() {
+  const [selectedGenres, setSelectedGenres] = useState<Genre[]>([]);
+  const navigate = useNavigate();
+
+  const handleGenreClick = (genre: Genre) => {
+    setSelectedGenres((prev) =>
+      prev.includes(genre) ? prev.filter((g) => g !== genre) : [...prev, genre]
+    );
+  };
+
+  return (
+    <div className="min-h-screen bg-dark flex flex-col p-4">
+      <div className="flex items-center justify-between mb-4">
+        <button onClick={() => navigate("/")} className="text-white text-xl">
+          <IoIosArrowBack />
+        </button>
+        <h1 className="text-white text-xl font-bold">Genres</h1>
+        <div></div>
+      </div>
+      <div className="grid grid-cols-2 gap-4 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-6 place-items-center">
```

Comment:
Please just follow Figma-Design for now. Grid with 4x4 cols, also on small display.